routes/orgs/[owner]/repos.ts

```diff
@@ -1,9 +1,16 @@
 import type { GithubRepo } from "~types";
 
 export default eventHandler(async (event) => {
-  // TODO: Do pagination
-  const rawRepos = await ghFetch(
-    `orgs/${event.context.params.owner}/repos?per_page=100`,
+  const query = getQuery(event);
+  const page = query.page ? Number(query.page) : 1;
+  const perPage = query.perPage ? Number(query.perPage) : 100;
```

Comment:
I think we can handle the pagination once and return theall pages instead of asking API consumers to do it.